ui/src/app-bar/app-bar.scss

```diff
@@ -62,4 +62,27 @@ $appbar-height: 48px;
             border-bottom-color: $color-blue-light;
         }
     }
+
+    &__data-menu-button {
+        position: relative;
+    }
+
+    &__data-menu > .menu {
+        position: absolute;
+        z-index: 10;
+        background-color: $color-blue-darker;
+
+        .menu__item {
+            color: $color-white-default;
+        }
```

Comment:
Meidän Menu-komponentissa saattaisi olla vähän refaktoroitavaa, että näitä ei tarvisi asettaa tätä kautta. En kuitenkaan tehnyt vielä täällä

Reply:
Eikös se ole ihan hyvä että värit on css:ssä? Vai meinasitko jotain muuta? Tuo position ja z-index on ehkä vähän hassut joutua käyttävän komponentin puolella tekemään.

Reply:
Otin nyt nämä vaan suoraan nykyisestä, ja ajattelin että niillä nyt ei ole _niin_ väliä kun ovat pelkkää devaajatavaraa. Mutta joo, nää voisi kyllä kääntää ihan suomeksikin

Reply:
Nyt meillä menu-komponentilla on omat tyylinsä, joissa ei ole minkäänlaista parametrisaatiota. Niissä määritellään värit ja sensellaista, ja nyt täällä vähän niinkuin ylikirjoitetaan toisen komponentin juttuja, joka on mielestäni melko iffyä. Tähän käyttötilanteeseen varmaan riittäisi, että määriteltäisiin menulle jonkinlainen teemaparametri (light vs. dark), ja tää menu olisi sitten käytännössä dark moden mukainen.